Bwt/BwtProj/Bwt.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace BWT;
+
+/// <summary>
+/// Burrows Wheeler Transformation implementation and inversion of it
+/// </summary>
+public static class Bwt
+{
+    private const int charAmount = 256;
+    
+    /// <summary>
+    /// BWT based on building <see cref="SuffixArray"/> 
+    /// </summary>
+    public static Tuple<char[], int> BwTransformation(String inputString)
+    {
+        var inputLength = inputString.Length;
+        var inputSuffix = new SuffixArray(inputString).sufArray;
+        var bwt = new char[inputLength];
+        for (var i = 0; i < inputLength; i++)
+        {
+            var j = inputSuffix[i] - 1;
+            if (j < 0)
+                j += inputLength;
+            bwt[i] = inputString[j];
+        }
+        return new Tuple<char[], int>(bwt, Array.IndexOf(inputSuffix, 0));
```

Comment:
Ну и тут, соответственно,
```suggestion
        return (bwt, Array.IndexOf(inputSuffix, 0));
```